### PR TITLE
Fix the install command via helm 3.x version.

### LINKED ISSUE #165: Adapt the installation command via helm 3.X
## Enhancement

**What version of Kubernetes are you using?**

* `kubectl version`

```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.0", GitCommit:"2bd9643cee5b3b3a5ecbd3af49d09018f0773c77", GitTreeState:"clean", BuildDate:"2019-09-18T14:36:53Z", GoVersion:"go1.12.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.4", GitCommit:"224be7bdce5a9dd0c2fd0d46b83865648e2fe0ba", GitTreeState:"clean", BuildDate:"2019-12-11T12:37:43Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
```

* `helm version`
```
version.BuildInfo{Version:"v3.0.2", GitCommit:"19e47ee3283ae98139d98460de796c1be1e3975f", GitTreeState:"clean", GoVersion:"go1.13.5"}
```

**What did you do?**

When Installing Chaos Mesh with helm, I received the msg.

```
hoge@Azure:~/chaos-mesh$ helm install helm/chaos-mesh --name=chaos-mesh --namespace=chaos-testing
Error: unknown flag: --name
```

**What did you expect to see?**
The installation command would support the both helm version with 3.x and 2.x.

**What did you see instead?**

I tried the below when using helm 3.x.
It seems well.

```
kubectl create ns chaos-testing # helm3.x don't create ns automatically.
helm install chaos-mesh helm/chaos-mesh  --namespace=chaos-testing
```

### PR DESCRIPTION
**Pull request by endurancetest20380119**
### What problem does this PR solve? 
fixes #165 

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

**Comment by siddontang** [comment]
thanks @endurancetest20380119 

please sign the CLA.

Btw, can we work with helm 2.x after this change?

**Comment by endurancetest20380119** [comment]
>Btw, can we work with helm 2.x after this change?
I completely forgot that.
Certainly this change may not be usable with helm 2.X.
Until helm2 reached eof, I thought it was kinder to write both.

File: README.md [hunk 1]
```diff
@@ -64,14 +64,23 @@ kubectl get crd podchaos.pingcap.com
 * Install Chaos Mesh with Chaos Operator only
 
 ```bash
+# create namespace chaos-testing 
+kubectl create ns chaos-testing
```
Comment (zhouqiang-cl) [inline README.md]: Did we need to create ns first?

File: README.md [hunk 2]
```diff
@@ -64,14 +64,23 @@ kubectl get crd podchaos.pingcap.com
...
+# helm 2.X
 helm install helm/chaos-mesh --name=chaos-mesh --namespace=chaos-testing
+# helm 3.X
+helm install chaos-mesh helm/chaos-mesh --namespace=chaos-testing
```
Comment (zhouqiang-cl) [inline README.md]: Could you add a blink line here, @dcalvin how about your opinion

File: README.md (hunk 1 above)
Comment (mahjonp) [reply README.md]: Yes, according to https://helm.sh/docs/faq/#automatically-creating-namespaces, Helm 3 returns an error if the namespace does not exist.

Comment (mahjonp) [reply README.md]: could you modify the Helm version limitation on `Prerequisites` part? @endurancetest20380119

**Comment by mahjonp** [comment]
Helm 3 [removes Tiller](https://helm.sh/docs/faq/#removal-of-tiller), so we should add some comments when mentioning Tiller.

`3. Check whether helm tiller pod is running`

File: README.md (hunk 1 above)
Comment (zhouqiang-cl) [reply README.md]: if using helm2, if the namespace exists, did the create command will fail

Comment (mahjonp) [reply README.md]: No, Helm 2 will create namespaces automatically if the specified ns doesn't exist.

Comment (zhouqiang-cl) [reply README.md]: LGTM

Comment (endurancetest20380119) [reply README.md]: https://github.com/helm/helm/issues/5753
I'm sorry in words less than.
I'm not very convinced of this fix.

Comment (endurancetest20380119) [reply README.md]: @mahjonp @zhouqiang-cl 
I fixed the `Prerequisites` part.

**Comment by zhouqiang-cl** [comment]
@endurancetest20380119 would you like to sign cla please? seems you have two committers in this pr `training` and `openshift-vm-user`

**Comment by endurancetest20380119** [comment]
> @endurancetest20380119 would you like to sign cla please? seems you have two committers in this pr `training` and `openshift-vm-user`
Two committers are the same. It's me. 
I review my git config and see if you can fix the username to endurancetest20380119.

**Comment by endurancetest20380119** [comment]
uh... I'm sorry about that.

**Comment by endurancetest20380119** [comment]
I pushed extra meta information by git push -f. 
I apologize.
Would you recreate the PR?

**Comment by zhouqiang-cl** [comment]
I think it is better to create another pr and close this one @endurancetest20380119 😄

**Comment by endurancetest20380119** [comment]
@zhouqiang-cl 
I'm sorry. I learned many things about that.
I checked some precious comments. 
https://github.com/pingcap/chaos-mesh/pull/166#issuecomment-577000045

**Pull request closed without merging**